Dockerfile

```diff
@@ -12,7 +12,11 @@ ENV STREAMLIT_SERVER_PORT 8501
 
 # Expose port
 EXPOSE 8501
-
 # Run Streamlit app
-CMD ["streamlit", "run", "climsight.py"]
+# CMD ["streamlit", "run", "src/climsight/climsight.py","$ARG"]
```

Comment:
Please, just remove it, no need to keep it.

Reply:
Removed.  :smiley: